### PR TITLE
feat: add identity pallet to the runtime

### PR DESCRIPTION
**Pull request by cuteolaf**

### CODE REVIEW

**Comment by cuteolaf** [comment]
@S0AndS0 @shibshib @Rubberbandits @camfairchild

**Comment by camfairchild** [comment]
@unconst @shibshib what do you think? I think this may be a good feature

**Comment by cuteolaf** [comment]
I believe that the community will like this feature

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -553,6 +559,30 @@ impl pallet_preimage::Config for Runtime {
     type ByteDeposit = PreimageByteDeposit;
 }
 
+parameter_types! {
+	pub const BasicDeposit: Balance = (1) as Balance * 2_000 * 10_000_000 + (258 as Balance) * 100 * 1_000_000;
```
Comment (camfairchild) [inline runtime/src/lib.rs:563]: Why is the calculation like this? Is there some documentation that can be added?

File: runtime/src/lib.rs [hunk 2]
```diff
@@ -553,6 +559,30 @@ impl pallet_preimage::Config for Runtime {
...
+	pub const FieldDeposit: Balance = (0) as Balance * 2_000 * 10_000_000 + (66 as Balance) * 100 * 1_000_000;
+	pub const SubAccountDeposit: Balance = (1) as Balance * 2_000 * 10_000_000 + (53 as Balance) * 100 * 1_000_000;
+	pub const MaxSubAccounts: u32 = 100;
```
Comment (camfairchild) [inline runtime/src/lib.rs:566]: Same for here

File: runtime/src/lib.rs [hunk 3]
```diff
@@ -553,6 +559,30 @@ impl pallet_preimage::Config for Runtime {
...
+	type Currency = Balances;
+	type BasicDeposit = BasicDeposit;
+	type FieldDeposit = FieldDeposit;
+	type SubAccountDeposit = SubAccountDeposit;
+	type MaxSubAccounts = MaxSubAccounts;
+	type MaxAdditionalFields = MaxAdditionalFields;
+	type MaxRegistrars = MaxRegistrars;
+	type Slashed = ();
+	type ForceOrigin = MoreThanHalfCouncil;
+	type RegistrarOrigin = MoreThanHalfCouncil;
```
Comment (camfairchild) [inline runtime/src/lib.rs:582]: So then (currently) adding a registrar is a 2 of 3 from the triumvirate? 
Wonder if using the Validator set makes more sense? Or perhaps an additional council group can be formed.

Comment (camfairchild) [reply runtime/src/lib.rs:582]: @unconst @shibshib

**Review by camfairchild — CHANGES_REQUESTED** [review]
Left some comments. Thanks for the PR

File: runtime/src/lib.rs (hunk 1 above)
Comment (cuteolaf) [reply runtime/src/lib.rs:563]: This is from the polkadot runtime.
https://github.com/paritytech/polkadot/blob/d74c60b6489f31f437edb1a19de0d0f6b8ad432e/runtime/polkadot/src/lib.rs#L620

File: runtime/src/lib.rs (hunk 2 above)
Comment (cuteolaf) [reply runtime/src/lib.rs:566]: same
https://github.com/paritytech/polkadot/blob/d74c60b6489f31f437edb1a19de0d0f6b8ad432e/runtime/polkadot/src/lib.rs#L621-L622

File: runtime/src/lib.rs (hunk 3 above)
Comment (cuteolaf) [reply runtime/src/lib.rs:582]: this should be a team decision, i believe.

**Pull request closed without merging**